src/devices/Common/IIntegratedLed.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Drawing;
+
+namespace Iot.Device
+{
+    public abstract class IntegratedLed : IDisposable
```

Comment:
Can we rename it to something like `LedStrip`? (P.S. the filename also has `I` prefix and it should match the class name instead)